src/registry/file/file-tab.tsx

```diff
@@ -0,0 +1,263 @@
+import {FileComponentSchema} from '@open-formulieren/types';
+import {useFormikContext} from 'formik';
+import isEqual from 'lodash.isequal';
+import {useContext} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import useAsync from 'react-use/esm/useAsync';
+
+import {Checkbox, Component, NumberField, Select, TextField} from '@/components/formio';
+import {BuilderContext} from '@/context';
+
+// // lifted from formio's file component in 4.13.x
```

Comment:
TODO - use this or something similar to validate the file upload size limit against the server configuration value